index.html

```diff
@@ -10,10 +14,70 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <nav>
+        <img class="logo" src="img/karma-logo.svg" alt="logo" width="35"/>
+        
+        <ul>
+            <li><a href="#" class="active">Meet Karma</a></li>
+            <li><a href="#">How It Works</a></li>
+            <li><a href="#">Store</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#">Login</a></li>
+        </ul>
+        <img id="hamburger" src="img/menu-hamburger.svg" alt=""/>
+    </nav>
+    <header class="hero">
+           
+            <div class="hero-text-and-button">
+                <h1>Introducing Karma</h1>
+                <p>Bring WiFi with you, everywhere you go.</p>
+                <button>Learn More</button>
+            </div>
+    </header>
+  
+    <main>
+        <section>
+            <div>                
+                <h2>Everyone needs a little Karma</h2>
+            </div> 
+            <div class="section-items">
+                <div class="section-item">
+                    <img src="img/icon-devices.svg" alt="" />
+                    <h3>Internet for all devices</h3>
+                </div>
+                <div class="section-item">
+                    <img src="img/icon-coffee.svg" alt="" />
+                    <h3>Boost your productivity</h3>
+                </div>
+                <div class="section-item">
+                    <img src="img/icon-refill.svg" alt="" />
+                    <h3>Pay as You Go</h3>
+                </div>
+            </div>
+            
+        </section>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+        <section class="feature">           
+                <img src="img/homepage-feature.png" alt="laughing female using tablet" />       
+                
+                <div class="feature_text-and-button">   
+                    <blockquote><q><span>Wherever I am, I just don't worry about my connection anymore!</span></q></blockquote>                              
+                        <button>Get Karma today</button>                
+                </div>            
+        </section>
+        
+    </main>
+    
+    <footer>
+        <hr/> 
+        <p class="first-p">Join us on</p>
+        <div class="footer-icons">
+            <img src="img/twitter-icon.svg" alt="twitter" width="20"/>
```

Comment:
As discussed in class, these should be wrapped in a `div` to prevent the borded squashing the image.